project1-sps/script.js

```diff
@@ -1,4 +1,177 @@
+var userWinCount = 0;
+var pcWinCount = 0;
+var drawCount = 0;
+var usernameInputCount = 0;
+var username = 0;
+var message = 0;
+
+// counting rounds and win lose percentage
+var roundsPlayed = 0;
+var winLoseRoundsPlayed = 0;
+
+var chooseSPS = function () {
+  var randomNumber = Math.random() * 3;
+  var foodNumber = Math.floor(randomNumber);
+  return foodNumber + 1;
+};
+
+var usernameInputCheck = function (userSPS) {
```

Comment:
Can have better naming for functions - start with a verb - so `checkUsernameInput`